### ISSUE TITLE
Texture Interpolation "Closest" not exported

### ISSUE THREAD

**Issue by H4kor** [godotengine/godot-blender-exporter#389]
**OS:**
Linux

**Godot version:**
v3.2.3

**Blender version:**
2.91

**Issue description:**

When setting the interpolation of a texture to "Closest" this setting is not reflected in the exported scene, resulting in a blurry texture in Godot.

**Minimal reproduction project:**

![image](https://user-images.githubusercontent.com/35053/103141671-d2a7a280-46f7-11eb-94fb-7f22e9b0682e.png)

1. Create a Shader using a Image Texture (see image)
2. Set the Interpolation to "Closest"
3. Export to Godot
4. Open scene in Godot: textures are blurred/do not use closest interpolation

**Comment by rcorre** [comment]
@H4kor can you provide an example? I frequently use "closest" interpolation for low-poly texturing (e.g. https://rcorre.itch.io/modular-mech), and it seems to work fine.

You might need to change your texture import settings in godot. Make sure your texture imports with lossless compression, no mipmaps, and no filtering (I actually don't know what "filtering" does, as there's no docstring, but I've just found experimentally that I should have it turned off for low poly art):

![1609943832](https://user-images.githubusercontent.com/2496231/103780684-4eacbf00-5003-11eb-8add-151faae67481.png)

I'm not sure the exporter can/should do this, as is isn't necessarily responsible for exporting the png (it does automatically, but only if the texture doesn't already exist). Also, multiple materials might use the same texture, but with different interpolation settings.

**Comment by H4kor** [comment]
Yes, changing the import settings and reimporting the texture fixes it. Might not be a bug but an enhancement as this can be confusing to new users at having the interpolation being exported too would be more convenient.

**Comment by rcorre** [comment]
>  Mipmaps are generated by default

I think that's the behavior of Godot, not the exporter. I don't think the exporter generates `.import` files at all. Though I suppose we could start.

I was thinking this was infeasible, since interpolation (e.g. closest/linear) is per-material, but import settings (e.g. filter/mipmaps/compression) are per-texture. You could have materials with different interpolation using the same texture. But I suppose that's pretty rare, and won't look good anyways, so maybe we should just assume you'd never want that.

**Comment by RadiantUwU** [comment]
I am on Godot 4.2-rc1 and i can't find not even a single way to get this working.
